### ISSUE TITLE
Defaults in eclipse.ini

### ISSUE THREAD

**Issue by kebeda** [mbooth101/eclipse-flatpak#10]
The default memory allocation with the flatpak is:
`-Xms40m`
`-Xmx512m`
This is too low for several of my workspaces, can you adjust to:
`-Xms512m`
`-Xmx2048m`
Or even better, is there some way to override this? Ideally in the eclipse.ini, but I can't locate it in the flatpak?

**Comment by mbooth101** [comment]
Hi, this is a good point, yes I will adjust these values to something more sensible by default.

In the meantime, you may be able to override the defaults by starting Eclipse from the command line, for example:

$ flatpak run org.eclipse.Platform -vmargs -Xmx2048m

**Comment by kebeda** [comment]
Yes, thank you, this worked! For those that come across this problem later, I resolved this by copying the `org.eclipse.Platform.desktop` file from `/var/lib/flatpak/app/org.eclipse.Platform/...` to `~/.local/share/applications/org.eclipse.Platform.desktop`, which to my understanding automatically takes precedence over anything that appears in `$XDG_DATA_DIRS`. I then added the `vmargs ...` needed at the end of the Exec line in `~/.local/share/applications/org.eclipse.Platform.desktop` and I am able to start Eclipse using my DE and have the right settings applied.
